minion/frontend/utils.py

```diff
@@ -11,7 +11,25 @@
 DEFAULT_FRONTEND_CONFIG = {
     'backend-api': {
         'url': 'http://127.0.0.1:8383'
-    }
+    },
+    'login': {
+		'type': 'persona', # persona OR ldap
+
+		'ldap' : {
+			'uri': 'ldaps://ldap.server/', # URI of the LDAP server
```

Comment:
I am going to change how this loads so that it is actual JSON here instead of JSON-esque Python; this will allow you to simply copy the text out of this file into frontend.json.